### ISSUE TITLE
[a11y] Missing aria-required-children when running Accessibility Insights on GroupedList

### ISSUE THREAD

**Issue by EtienneBerube** [microsoft/fluentui#12274]
### Environment Information

- **Package version(s)**: 7.85.0
- **Browser and OS versions**: Windows 10 1909 and Edge  80.0.361.66

### Describe the issue:

When running [accessibility insights](https://accessibilityinsights.io/) on a page using GroupedList, the following error is shown:
![image](https://user-images.githubusercontent.com/18299036/76473249-26d5d600-63b5-11ea-8311-27380a4c616b.png)

There is no known work around for now. 

I am aware of this [issue] (https://github.com/OfficeDev/office-ui-fabric-react/issues/1071). But it dates from 2017 and I cannot reopen it as I am not a contributor.

### Please provide a reproduction of the issue in a codepen:

This will not work within codepen due to the rendering method. However, running the Accessibility Insight on the [documentation]( https://developer.microsoft.com/en-us/fabric#/controls/web/groupedlist) will give the same error.

#### Actual behavior:

Gives an error saying "aria-required-children is missing".

[Source](https://dequeuniversity.com/rules/axe/3.5/aria-required-children?application=msftAI) on why this should be implemented.  

#### Expected behavior:

The aria-required-children should be added to the component.

**Comment by aneeshack4** [comment]
@EtienneBerube Thanks for reporting this! I ran a FastPass on the GroupedList docs page and see the problem too: 
![image](https://user-images.githubusercontent.com/4161791/76558547-78d93300-646b-11ea-905a-ef9812112edd.png)
It looks like the GroupedList has `role="grid"`. According to the ARIA spec, the cells/children of a `grid` can technically still have a `role="row"`. I'm a little confused though because looking at the `GroupedList.base.tsx` code, I see no `role="grid"` anywhere only `role="presentation"` https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.base.tsx#L109. Tagging owners @aditima @KevinTCoughlin - do y'all know why GroupedList wasn't originally given a role of `rowgroup` for the parent and `row` for children? And where this `grid` role is coming from when `presentation` is actually set in code? I see that `DetailsList.base.tsx` is the only other relevant place that has `role="grid"` set but I don't see that being directly used in the GroupedList code...

**Comment by khmakoto** [comment]
This has been solved by #13457.
